lesson07/assignment/hello-world/src/main.js

```diff
@@ -0,0 +1,22 @@
+import { createRouter, createWebHistory } from 'vue-router'
+import App from './App.vue'
+import HomePage from './components/HomePage.vue'
+import HistoryPage from './components/HistoryPage.vue'
+import ComparisonPage from './components/ComparisonPage.vue'
+import DrinksPage from './components/DrinksPage.vue'
+
```

Comment:
Don't forget to import bootstrap.

`import 'bootstrap'
import 'bootstrap/dist/css/bootstrap.css'
import 'bootstrap-vue/dist/bootstrap-vue.css'`